### DISCUSSION TITLE
6

### DISCUSSION THREAD

**Discussion by opqphp** [share121/inter-knot#3132] [其他]
<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >
![Screenshot_2024-08-19-20-07-40-84_a87fd7db6caa850b517aa6fa9d2fcd0e](https://github.com/user-attachments/assets/7d3e61c2-ec4f-460c-9964-e7adb223ef03)

**Comment by opqphp** [comment]
#3114
![Screenshot_2024-08-19-20-07-40-84_a87fd7db6caa850b517aa6fa9d2fcd0e](https://github.com/user-attachments/assets/dd42112c-1d65-41be-ad32-0d81ba8f93cd)

**Comment by YO0oi** [comment]
6
